### PR TITLE
Proposal of nicer structure of Readme

### PR DESCRIPTION
**Pull request by timgrossmann**
- Increase readability
- highlight more important parts
- space out content to avoid cluttering 

---

Let me know what you think

@anoff Ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,72 @@
...
-[![Build Status](https://github.com/anoff/devradar/workflows/web/badge.svg)](https://github.com/anoff/devradar/actions)
-
-[<img src="assets/logo-text.png">](https://devradar.io)
-
-> Visit [devradar.io](https://devradar.io) for an introduction
+<p align="center">
+  <img src="assets/logo-text.png">
+  <p align="center"><b>Track and manage</b> your and your teams skill set to <b>accelerate progress and advance productivity</b>.<p>
+  <p align="center">
+    <a href="./LICENSE">
```
Comment (anoff) [inline README.md]: Not really a fan of HTML inside markdown, unless neessary.
The center tags are OK but why replace links and image tags?

File: README.md [hunk 2]
```diff
@@ -1,44 +1,72 @@
...
+* [Web](web/): devradar landing page at [devradar.io](//devradar.io) (Hugo)
 
 See [anoff/devradar-static](https://github.com/anoff/devradar-static) for an easy to setup version of the devradar to present your own tech skills.
 
+<br />
+
 ## Demo
 
-### devradar editor
+### Devradar editor
```
Comment (anoff) [inline README.md]: logo (brand) is lowercase

File: README.md [hunk 3]
```diff
@@ -1,44 +1,72 @@
...
-[![Build Status](https://github.com/anoff/devradar/workflows/editor/badge.svg)](https://github.com/anoff/devradar/actions)
-[![Build Status](https://github.com/anoff/devradar/workflows/teams/badge.svg)](https://github.com/anoff/devradar/actions)
-[![Build Status](https://github.com/anoff/devradar/workflows/web/badge.svg)](https://github.com/anoff/devradar/actions)
-
-[<img src="assets/logo-text.png">](https://devradar.io)
-
-> Visit [devradar.io](https://devradar.io) for an introduction
+<p align="center">
+  <img src="assets/logo-text.png">
+  <p align="center"><b>Track and manage</b> your and your teams skill set to <b>accelerate progress and advance productivity</b>.<p>
```
Comment (anoff) [inline README.md]: Would rather focus on **individual/team competences** than progress/productivity

File: README.md [hunk 4]
```diff
@@ -1,44 +1,72 @@
...
+    <a href="https://github.com/anoff/devradar/actions">
+	    <img src="https://github.com/anoff/devradar/workflows/web/badge.svg">
+    </a>
+  </p>
+</p>
+
+
+<br />
+
+**Get more information: [Try out devradar.io here](https://devradar.io)**
```
Comment (anoff) [inline README.md]: Love the catchy twist but the devradar.io page can not be "tried out".
Imho should either point to editor or replace *try* with *learn more* ...

File: README.md (hunk 1 above)
Comment (timgrossmann) [reply README.md]: The reason for this is that it nesting Markdown inside HTML does not work. 😅

File: README.md (hunk 3 above)
Comment (timgrossmann) [reply README.md]: What text would you use?

File: README.md (hunk 4 above)
Comment (timgrossmann) [reply README.md]: True, how about both?

File: README.md (hunk 3 above)
Comment (anoff) [reply README.md]: Track and manage skills as an individual and manage competences across a team.

File: README.md (hunk 4 above)
Comment (anoff) [reply README.md]: Not sure what you mean... Could be links to both if you mean that.
Just don't thing "trying out" and the about page fits together word-wise

File: README.md (hunk 2 above)
Comment (timgrossmann) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (timgrossmann) [reply README.md]: Much better.

File: README.md (hunk 4 above)
Comment (timgrossmann) [reply README.md]: Put both let me know if that works

**Comment by timgrossmann** [comment]
@anoff Ready for review

**Pull request merged**